### ISSUE TITLE
pre-softmax logits for graphs as graph representations

### ISSUE THREAD

**Issue by paulamartingonzalez** [dmlc/dgl#1656]
## 📚 Documentation

Could you please provide the code to reproduce the pre-softmax logits for graphs as graph representations presented in https://docs.dgl.ai/tutorials/basics/4_batch.html? It could be really useful.

**Comment by mufeili** [comment]
Hi, what do you mean by providing the code to reproduce the pre-softmax logits? I think it should already be there.

**Comment by paulamartingonzalez** [comment]
Thanks for your reply! The image is already there but there is no code to get it. If you could share the code used to create in order to reproduce the results it would be great, thanks!

**Comment by mufeili** [comment]
I think you are able to directly copy the code like below?

```python
import torch.nn as nn
import torch.nn.functional as F

class Classifier(nn.Module):
    def __init__(self, in_dim, hidden_dim, n_classes):
        super(Classifier, self).__init__()
        self.conv1 = GraphConv(in_dim, hidden_dim)
        self.conv2 = GraphConv(hidden_dim, hidden_dim)
        self.classify = nn.Linear(hidden_dim, n_classes)

    def forward(self, g):
        # Use node degree as the initial node feature. For undirected graphs, the in-degree
        # is the same as the out_degree.
        h = g.in_degrees().view(-1, 1).float()
        # Perform graph convolution and activation function.
        h = F.relu(self.conv1(g, h))
        h = F.relu(self.conv2(g, h))
        g.ndata['h'] = h
        # Calculate graph representation by averaging all the node representations.
        hg = dgl.mean_nodes(g, 'h')
        return self.classify(hg)
```

The output of ```self.classify(hg)``` are pre-softmax logits.
